### DISCUSSION TITLE
Xamarin support

### DISCUSSION THREAD

**Discussion by drwill-ms** [Azure/azure-iot-sdk-csharp#2103] [General]
This will be the official discussion thread for Xamarin support. I'll link existing issues to this discussion and close them down.

The latest status is that we've had an engineer run through scenarios to see what the gaps are, as well as look at each reported issue. We're noting the gaps so we can determine when we can commit to doing the work to close them, and offer official support for the platform.

**Comment by drwill-ms** [comment]
Linked issues #1306, #563, #561, #557, and #2190.

**Reply by rido-min** [reply]
I've found this sample in the `main` branch of  our samples repo.

https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/iot-hub/Samples/device/XamarinSample

It's 3 years old, and links to this GH project/board with all tasks in the "done" column. somehow confusing. 
https://github.com/Azure/azure-iot-sdk-csharp/projects/1 

Should we add the linked issues to that board? or just delete it?

**Comment by drwill-ms** [comment]
Note: x509 authenticated Xamarin apps won't work over HTTP. The class is unimplemented in Mono.

<https://bugzilla.xamarin.com/23/23075/bug.html>
